aiida/common/log.py

```diff
@@ -52,6 +53,8 @@ def report(self, msg: str, *args, **kwargs) -> None:
     logging.getLevelName(logging.CRITICAL): logging.CRITICAL,
 }
 
+LogLevels = enum.Enum('LogLevels', {key: key for key in LOG_LEVELS})  # type: ignore[misc]
```

Comment:
Don't think so, this is the error:
```
Enum() expects a string, tuple, list or dict literal as the second argument  [misc]
```
I am not sure why `{key: key for key in LOG_LEVELS}` is not considered a dict literal though.

Reply:
This just seems to be a shortcoming of `mypy` that cannot determine the type of the dictionary statically: https://github.com/python/mypy/issues/5317

Reply:
Okay, then all good. I nitpick it because I didn't find anything to comment on the first quick go through of your implementation, which makes me seem not very professional:sunglasses: